src/modules/10-common/hooks/useModuleLicenses.tsx

```diff
@@ -12,9 +12,7 @@ import { useModuleInfo } from './useModuleInfo'
 export function useAnyEnterpriseLicense(): boolean {
   const { licenseInformation } = useLicenseStore()
 
-  const anyEntitlement = Object.values(licenseInformation).find(
-    license => license?.edition === Editions.ENTERPRISE && license?.status === 'ACTIVE'
-  )
+  const anyEntitlement = Object.values(licenseInformation).find(license => license?.edition === Editions.ENTERPRISE)
```

Comment:
You can simplify this if you swap from a `find` to a `some` and just return it.
```suggestion
  return Object.values(licenseInformation).some(license => license?.edition === Editions.ENTERPRISE)
```